wfe2/wfe_test.go

```diff
@@ -2018,6 +2182,44 @@ func TestGetCertificateHEADHasCorrectBodyLength(t *testing.T) {
 	test.AssertEquals(t, 0, len(body))
 }
 
+type mockSAWithError struct {
+	core.StorageGetter
+}
+
+func (sa *mockSAWithError) GetCertificate(_ context.Context, serial string) (core.Certificate, error) {
+	return core.Certificate{}, errors.New("Oops")
+}
+
+func TestGetCertificateServerError(t *testing.T) {
+	// TODO: add tests for failure to parse the retrieved cert, a cert whose
+	// IssuerNameID is unknown, and a cert whose signature can't be verified.
```

Comment:
If my prior postulation proves correct, I believe this can go.

Reply:
This is just noting "there are error paths in the function under test which aren't tested here". Since we have those error paths, we should have test cases which cover them.